vscode-antimony/src/server/stibium/test/import-test-data/func.ant

```diff
@@ -0,0 +1,5 @@
+import "import_func.ant"
+
+function varsum(a, b)
+  a + b
+end
```

Comment:
This file should fail, because it defines the 'varsum' function twice.  Does it?

What I would also be interested in seeing is a file like:

import "import_func.ant"

a = varsum(b, c)


i.e. the function is defined in the imported file, and used by the importing file.

Reply:
The varsum function being defined twice is tested in the test_dupe_func method, which makes sure there's an error in the imported file since varsum is being defined twice. Also added the aforementioned test case to test_func_call